jax/_src/sharding_impls.py

```diff
@@ -729,7 +729,7 @@ class NonUniformShardingError(ValueError):
 
 @util.cache(max_size=4096, trace_context_in_key=False)
 def get_process_index_and_count(
```

Comment:
This is a private API, not a public API.

So I am not sure if this change is ideal.

Reply:
Yes